### PR TITLE
Fix buggy loop condition + watch pod clusters in p2-watch

### PR DESCRIPTION
**Pull request by anthonybishopric**

### CODE REVIEW

**Comment by mpuncel** [comment]
👍

File: bin/p2-watch/main.go [hunk 1]
```diff
@@ -32,32 +41,77 @@ func main() {
...
+		podCh := make(chan []kp.ManifestResult)
+		go store.WatchPods(podPrefix, types.NodeName(*nodeName), quit, errChan, podCh)
+		for {
+			select {
+			case results := <-podCh:
+				if len(results) == 0 {
+					fmt.Println(fmt.Sprintf("No manifests exist for %s under %s (they may have been deleted)", *nodeName, podPrefix))
+				} else {
+					for _, result := range results {
+						fmt.Println("")
```
Comment (petertseng) [inline bin/p2-watch/main.go]: I realize this is just an indentation change, but I am a bit curious - what was this used for?

Comment (petertseng) [inline bin/p2-watch/main.go]: oh I guess it makes it easier to understand where one output ends and one begins

Comment (anthonybishopric) [inline bin/p2-watch/main.go]: i'm pretty sure the `fmt.Println` is a mistake and I'll remove it

**Pull request merged**